tests/platform_helper/providers/test_schema_migrator.py

```diff
@@ -0,0 +1,233 @@
+from unittest.mock import create_autospec
+
+import pytest
+import yaml
+
+from dbt_platform_helper.providers.io import ClickIOProvider
+from dbt_platform_helper.providers.schema_migrator import (
+    InvalidMigrationConfigurationException,
+)
+from dbt_platform_helper.providers.schema_migrator import Migrator
+from dbt_platform_helper.providers.version import InstalledVersionProvider
+
+
+class MockMigration:
+    def __init__(self, name, from_version, call_record_list):
+        self.name = name
+        self._from_version = from_version
+        self.call_record_list = call_record_list
+
+    def from_version(self) -> int:
+        return self._from_version
+
+    def to_version(self) -> int:
+        return self._to_version
```

Comment:
It does not have a to version.

Reply:
It did in one iteration. Missed removing this one 👍